### DISCUSSION TITLE
Already started the trial and updated the billing information but still not authorized

### DISCUSSION THREAD

**Discussion by peterAzv** [community/community#21349] [Copilot Conversations]
As the title says.

I already updated my subscription to trial, and my billing information, everything seems correct in the github page, but when I try to use the copilot in visual studio code, its says "Invalid copilot token: missing token: 403 " in the output.

**Comment by DrWarmoose** [comment]
Exact same problem too.  Appears that this has been left unanswered for over a month.
Paying for a service that won't work on any of my IDEs and zero support.
